tests/test_playbooks/host_power.yml

```diff
@@ -1,32 +1,80 @@
 ---
+- hosts: fixtures
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  vars:
+    host_name: vm01.example.com
+  tasks:
+  - include: tasks/host_power.yml
+    vars:
+      host_power_name: "{{ host_name }}"
+      host_power_state: "off"
+
 - hosts: tests
   gather_facts: false
   vars_files:
     - vars/server.yml
+  vars:
+    host_name: vm01.example.com
   tasks:
   - include: tasks/host_power.yml
     vars:
+      host_power_name: "{{ host_name }}"
+      host_power_state: "state"
+      expected_change: false
+      expected_power_state: "off"
+
+  - include: tasks/host_power.yml
+    vars:
+      host_power_hostname: "{{ host_name }}"
       host_power_state: "state"
       expected_change: false
       expected_power_state: "off"
 
   - include: tasks/host_power.yml
     vars:
+      host_power_name: "{{ host_name }}"
       host_power_state: "on"
       expected_change: true
 
   - include: tasks/host_power.yml
     vars:
+      host_power_name: "{{ host_name }}"
       host_power_state: "on"
       expected_change: false
 
   - include: tasks/host_power.yml
     vars:
+      host_power_name: "{{ host_name }}"
+      host_power_state: "start"
+      expected_change: false
+
+  # Sometime host power state change is not reflected instantaneously
+  # and make this playbook fail when recording tests
+  - pause:
+      seconds: 5
```

Comment:
I think in syc, there is an example for not waiting while not recording.

Reply:
@mdellweg, thanks for the pointer !